frontend/package.json

```diff
@@ -6,8 +6,9 @@
     "@testing-library/jest-dom": "^5.17.0",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
-    "axios": "^1.5.0",
-    "random-words": "^2.0.0",
+
```

Comment:
Add "axios" and "random-words" packages also. Don't remove those. You can delete your node modules in the local machine and add these two lines to package.json
```
"axios": "^1.5.0",
"random-words": "^2.0.0",
"font-awesome": "^4.7.0",
```
Then `npm install` again in your local